app/models/project_group.rb

```diff
@@ -62,6 +62,11 @@ def summary_or_none
     summary.blank? ? "[#{I18n.t("none")}]" : summary.to_s
   end
 
+  # For the "Group" dropdown in step edit form
+  def option_label
+    ("&nbsp; &nbsp; " * [depth - 1, 0].max).html_safe + summary_or_none
```

Comment:
It's more efficient to use `<<` instead of `+` in this case b/c it appends to the string instead of creating a whole new one.